binary/typedb.bat

```diff
@@ -75,6 +76,20 @@ if exist .\server\ (
   goto exiterror
 )
 
+:startcluster
+
+set "G_CP=%TYPEDB_HOME%\server\conf\;%TYPEDB_HOME%\server\lib\common\*;%TYPEDB_HOME%\server\lib\prod\*"
+
+if exist .\server\ (
+  java %SERVER_JAVAOPTS% -cp "%G_CP%" -Dtypedb.dir="%TYPEDB_HOME%" -Dtypedb.conf="%TYPEDB_HOME%\%TYPEDB_CONFIG%" com.vaticle.typedb.cluster.server.TypeDBClusterServer %2 %3 %4 %5 %6 %7 %8 %9
+  goto exit
+) else (
+  echo TypeDB Cluster is not included in this TypeDB distribution^.
+  echo You may want to install TypeDB Cluster or TypeDB ^(all^)^.
```

Comment:
`...or TypeDB Cluster ^(all^)^.`

Reply:
@lolski addressed in https://github.com/vaticle/typedb-common/pull/112/commits/1b57e4cca184f3089942f8ed24a641dfdd5ee47c